.github/actions/deploy-documentdb/action.yml

```diff
@@ -0,0 +1,195 @@
+name: 'Deploy DocumentDB Cluster'
+description: 'Deploys a DocumentDB cluster and waits for it to be ready'
+inputs:
+  architecture:
+    description: 'Target architecture for logging purposes'
+    required: true
+  namespace:
+    description: 'Kubernetes namespace for the DocumentDB cluster'
+    required: true
+  cluster-name:
+    description: 'Name of the DocumentDB cluster'
+    required: true
+  node-count:
+    description: 'Number of nodes in the cluster'
+    required: false
+    default: '1'
+  instances-per-node:
+    description: 'Number of instances per node'
+    required: false
+    default: '1'
+  documentdb-image:
+    description: 'DocumentDB container image to use'
+    required: false
+    default: 'ghcr.io/microsoft/documentdb/documentdb-local:16'
+  pvc-size:
+    description: 'Size of persistent volume claim'
+    required: false
+    default: '5Gi'
+  public-load-balancer:
```

Comment:
I talked with Rayhan and we always want an LB but IP should be private/public based on Cx need

Reply:
Yes, PR is updated and approved. will be merged soon. https://github.com/microsoft/documentdb-kubernetes-operator/pull/39/files